source/funkin/ui/freeplay/FreeplayState.hx

```diff
@@ -2073,6 +2077,9 @@ class FreeplayState extends MusicBeatSubState
           restartTrack: false
         });
       FlxG.sound.music.fadeIn(2, 0, 0.8);
+
+      Conductor.instance.mapTimeChanges([new SongTimeChange(0, 145)]);
```

Comment:
What about something like this instead of using a hardcoded value?

```suggestion
      var songMusicData:Null<SongMusicData> = SongRegistry.instance.parseMusicData('freeplayRandom');
      songBPM = songMusicData.timeChanges[0].bpm;
      Conductor.instance.mapTimeChanges([new SongTimeChange(0, songBPM)]);
```
So mods that change the Random track have their BPM applied properly.

Reply:
a changeable file for that would be good but as of right now it doesn't exist which is why I just hard coded the bpm